src/pages/PrescriptionPage.js

```diff
@@ -11,27 +11,21 @@ import PropTypes from 'prop-types';
 import { View } from 'react-native';
 import { connect } from 'react-redux';
 
-import {
-  FavouriteStarIcon,
-  TableShortcut,
-  TableShortcuts,
-  Wizard,
-  SimpleTable,
-  PageButton,
-} from '../widgets';
+import { Wizard, SimpleTable, PageButton } from '../widgets';
 import { PrescriptionCart } from '../widgets/PrescriptionCart';
 import { PrescriptionSummary } from '../widgets/PrescriptionSummary';
 
 import { UIDatabase } from '../database';
 import { getColumns } from './dataTableUtilities';
-import { ALPHABET } from '../widgets/constants';
```

Comment:
yup - sorry should have mentioned it.

My thoughts were:
Have removed it as it's turned out to be slightly more complicated than initially thought. Just in regards to the keyboard settings we use (which reduce the size of the viewport, rather than overlapping) meaning might have to do a few things with scroll views and absolute dimensions. I don't think it's really too difficult but am just trying to reduce the functionality and get this into develop with a basic but working implementation and then iterate on it from there and decided this was gonna get cut for now! 

Let me know if you agree or not!